.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     container:
-      image: quay.io/fedora/fedora:35
+      image: quay.io/fedora/fedora:36
```

Comment:
Latest stable is 37.

Alternatively may be interesting to use quay.io/centos/centos:stream9 here e.g.

Reply:
i'm fine with either, just wanted to get things working again

Reply:
should this be updated to 37?